discordbot/cogs/internet.py

```diff
@@ -3,16 +3,24 @@
 """
 
 from random import randint
+from selenium import webdriver
 
+import wget
+import json
 import aiohttp
 import discord
 import pyowm
+import os
+import requests
 from bs4 import BeautifulSoup as BSoup
 from discord.ext import commands
 
 from discordbot.bot import DiscordBot
 from discordbot.consts import bot_config
 
+depot = DepotManager.get()
+driver = webdriver.PhantomJS()
```

Comment:
`PhantomJS` requires node to install, and is also incompatible with `http://ripme.xyz/`